bin/polykey.ts

```diff
@@ -0,0 +1,373 @@
+#!/usr/bin/env node
+
+// this will be a CLI entrypoint to this library
+// we expect that this can be used as a CLI
+// application directly as js-polykey
+
+import { program } from 'commander'
+import inquirer from 'inquirer'
+import PolyKey from '../src/Polykey'
+import chalk from 'chalk'
+import os from 'os'
+import fs from 'fs'
+
+let pk: PolyKey
+// Write a `file`
+let tempDir: string
+/*******************************************/
+// initialization
+function initPolyKey(
+    password: string | null,
+    polykeyPath: string | undefined = undefined,
+    publicKeyPath: string | undefined = undefined,
+    privateKeyPath: string | undefined = undefined,
+    privatePassphrase: string | undefined = undefined,
+    verbose: boolean | undefined = undefined
+) {
+    try {
+        // Initialize polykey
+        if (password !== null) {
+            pk = new PolyKey(Buffer.from(password), undefined, undefined, polykeyPath)
```

Comment:
yeah we could do that but I've realised there are only two things that need to be parameters; the KeyManager and the home directory. Every thing else was moved to the KeyManager itself.

Reply:
I've left it as just standard non-config object style parameters, let me know if you think we should have the config object and I will put it in. I left it out because I reduced the parameters required to 2

Reply:
Ok that's fine.